### ISSUE TITLE
ros install arrival :  READ error during COMPILE-FILE:

### ISSUE THREAD

**Issue by aymeric75** [roswell/roswell#551]
Hello, 

I have an issue when running: `ros install arrival` ,  see below:

## Environment

```
# result of evaluating
uname -a
Linux c9ff33679193 5.4.0-144-generic #161-Ubuntu SMP Fri Feb 3 14:49:04 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux
ros --version    # if roswell runs successfully
roswell 22.12.14.113(eae6e5c)
ros config

setup.time=3889156857
sbcl-bin.version=2.3.3
default.lisp=sbcl-bin
```

We may assign explanatory tags to bad questions instead of replying to it.
See this famous text on how to ask good questions http://catb.org/~esr/faqs/smart-questions.html

## How to reproduce the issue

```sh

ros install arrival

```

## Actual result

```sh

To load "arrival":
  Load 1 ASDF system:
    arrival
; Loading "arrival"
..[1/3] System 'arrival' found. Loading the system..
; 
; caught ERROR:
;   READ error during COMPILE-FILE:
;   
;     Lock on package SB-C violated when interning LAMBDA-PARENT while in package
;     LOG4CL-IMPL.
;   See also:
;     The SBCL Manual, Node "Package Locks"
;   
;     (in form starting at line: 99, column: 0, position: 3779)
Aborted during step [1/3].
Unhandled UIOP/LISP-BUILD:COMPILE-FILE-ERROR in thread #<SB-THREAD:THREAD tid=74280 "main thread" RUNNING
                                                          {1004498113}>:
  COMPILE-FILE-ERROR while
  compiling #<CL-SOURCE-FILE "log4cl" "src" "naming-sbcl">


```

**Comment by aymeric75** [comment]
Hello, I might have found the source of the error, when installing ROS it appears that it will also install SBCL, but it installs v2.3.3, and all this was working before with version 2.3.2. So how can I have ROS  install the right version of SBCL ?

**Comment by snmsts** [comment]
right version? ``ros install sbcl-bin/2.3.2``.

I guess it's better to plan fix log4cl for sbcl 2.3.3.

**Comment by snmsts** [comment]
``ros checkout log4cl`` will do for sbcl 2.3.3 till next quicklisp update.
